.github/workflows/kani.yml

```diff
@@ -35,4 +35,11 @@ jobs:
           submodules: true
 
       - name: Run Kani Script
-        run: bash ./head/scripts/check_kani.sh ${{github.workspace}}/head
+        run: head/scripts/run-kani.sh --path ${{github.workspace}}/head
+
+      - name: Run Kani Script with --kani-args and -p
+        run: head/scripts/run-kani.sh -p ${{github.workspace}}/head --kani-args --harness ptr::verify::check_read_u128 --harness ptr::unique::verify::check_as_mut
```

Comment:
I would suggest that you change the arguments to be something that doesn't depend on the harnesses configuration, or something very generic, like `--harness ptr`, which is less likely to be broken if we rename a harness.